### PR TITLE
[warning] in tests

### PR DESCRIPTION
**Pull request by alexmarie78**
Tried to fix : Warning in tests #1091

### CODE REVIEW

File: pyFAI/test/test_goniometer.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def test_goniometer(self):
         g = Goniometer([1., 2., 3., 4., 5., 6., 7.], self.gt, "pilatus100k")
         fname = os.path.join(UtilsTest.tempdir, "gonio.json")
         g.save(fname)
-        self.assert_(os.path.exists(fname), "json file written")
+        self.assertTrue(os.path.exists(fname))
```
Comment (kif) [inline pyFAI/test/test_goniometer.py]: why did you remove the comment on the error ?

Comment (alexmarie78) [reply pyFAI/test/test_goniometer.py]: I looked upon assertTrue documentation, and I thought that if it is true, the comment on the error would not be printed. My mistake sorry..

Comment (vallsv) [reply pyFAI/test/test_goniometer.py]: Usually comment are useless. That's time consuming for nothing. You anyway have to go back to the code.

For example here, the message should be "json file not written"

**Comment by vallsv** [comment]
LGTM
